components/JEDI/JEDINav.js

```diff
@@ -0,0 +1,41 @@
+import Link from 'next/link';
+import React from 'react';
+
+import { blogLink } from '../../data/jedi';
+import styles from '../../styles/pages/jedi.module.scss';
+
+const links = [
+  {
+    path: '/jedi',
+    title: 'home',
+  },
+  {
+    path: '/jedi/meet',
+    title: 'meet the JEDIs',
+  },
+  {
+    path: '/jedi/allyship',
+    title: 'allyship spaces',
+  },
+  {
+    path: blogLink,
+    title: 'blog',
+    ext: true,
+  },
+];
```

Comment:
personal preference: have the `ext` flag set to false instead of undefined for the internal pages

Reply:
Yup, I can do that.